src/pages/neighborhoods/neighborhood.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Box,
+  Circle,
+  Heading,
+  HStack,
+  Icon,
+  Stack,
+  Tag,
+  Text,
+  WrapItem,
+} from "@liftedinit/ui";
+import { RiHome2Line } from "react-icons/ri";
+
+export interface NeighborhoodInfo {
+  name: string;
+  description: string;
+  services: string[];
+}
+
+function pickColor(str: string) {
+  const hue = str
+    .split("")
+    .reduce((acc, _, i) => str.charCodeAt(i) + ((acc << 6) - acc), 0);
+  return `hsl(${hue % 360}, 100%, 66%)`;
+}
```

Comment:
This little beauty will generate a random color determined by the title of the neighborhood, e.g. "Alpha Ledger" will always get the same color.

Reply:
What's the complete color range? Are there colors we don't want, e.g., because it melts with the background?

Reply:
I adapted it from this: https://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript

We're only hashing and then modulo-ing the hue, so saturation and value will remain constant and we'll get nice vibrant but dark colors. They should always stand out against the background.